terraform/README.md

```diff
@@ -0,0 +1,59 @@
+# Terraform
+
+## Get started
+
+1. Set up a service key
+    ```bash
+    # login
+    cf login -a api.fr.cloud.gov --sso
+    # follow temporary authorization code prompts
+    # select org "gsa-sandbox", and your space
+
+    # create a service instance that can provision service accounts
+    # the value for < YOUR-NAME > can be any version of your name, it isn't significant
+    cf create-service cloud-gov-service-account space-deployer < YOUR-NAME >
+
+    # bind a service key to the service instance
+    cf create-service-key < YOUR-NAME > space-deployer-key
+
+    # return a username/password pair for the service instance
+    cf service-key < YOUR-NAME > space-deployer-key
+    ```
+1. Copy `terraform/<ENV>/secrets.tfvars.example` to `secrets.tfvars` and add the service key information
+    ```
+    cf_user = "some-dev-user"
+    cf_password = "some-dev-password"
+    ```
+1. Run terraform with
+    ```bash
+    terraform init
+    terraform plan
+    terraform apply
+    ```
+1. Assuming all is well, now you can bind RDS to your app
+    ```bash
+    cf bind-service < YOUR-APP > < YOUR-RDS-SERVICE >
+    cf restage < YOUR-APP >
+    ```
+
+## Structure
+
+Each environment has its own module, which relies on a shared module for everything except the providers code and environment specific variables and settings.
+
+```
+- database_<env>/
```

Comment:
[question] if we have different services, are we planning on splitting them into their own module? or having them all under `/dev`? i am just wondering if we need the `database_` namespacing.

Reply:
I would be happy to organize this however we want.  I was taking inspiration from the ECLKC vendor team, which organized them like `service_env` and then `modules/service`, but I think it makes sense to have `env/service` instead.